Documentation/operations/upgrade.rst

```diff
@@ -223,7 +219,7 @@ configuration options for each minor version.
 
 .. note::
 
-   The above variant can not be used in combination with ``--set`` or providing
+   The above variant can not be used in combination with ``--set `` or providing
```

Comment:
there's a white space after `--set` that shouldn't be there.

Reply:
fixd